Library/Homebrew/utils/pypi.rb

```diff
@@ -35,13 +33,16 @@ def initialize(package_string, is_url: false)
         return
       end
 
-      @name = package_string
-      @name, @version = @name.split("==") if @name.include? "=="
+      if package_string.include? "=="
+        @name, @version = package_string.split("==")
+      else
+        @name = package_string
+      end
```

Comment:
This is a slight rearrangement to reduce the number of `T.must` calls that would otherwise be necessary.